README.md

```diff
@@ -6,6 +6,21 @@ tray icons without having to deal with a graphical toolkit like GTK.
 `mktrayicon` can be used two ways: To create an icon that is controlled by a 
 named pipe or, more simply, to create a non-interactive icon.
 
+Calls to mktrayicon are flexible. All of these work:
```

Comment:
On second thought, I feel like this list probably isn't useful. Instead, I think it's sufficient to say that "if you do not pass in a pipe, `mktrayicon` will run with the given icon and tooltip text until terminated".